docs/peripheral.md

```diff
@@ -32,3 +40,4 @@ added to the end of the name of peripherals that only have a single instance to
 differentiate the peripheral name and the instance name.
 The following peripheral instances are defined (listed alphabetically):
 - `::picolibrary::Arm::Cortex::M0PLUS::Peripheral::NVIC0`
+- `::picolibrary::Arm::Cortex::M0PLUS::Peripheral::SYSTICK0`
```

Comment:
Add note that this peripheral instance is only available if `PICOLIBRARY_ARM_CORTEX_M0PLUS_IMPLEMENTATION_HAS_SYSTICK` is true.